.eslintrc

```diff
@@ -0,0 +1,102 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
+  "plugins": [
+    "flow-vars",
+    "react",
+    "jsx-a11y",
+    "import",
+    "babel",
+    "immutable"
+  ],
+  "env": {
+    "browser" : true
+  },
+  "rules": {
+    "semi" : [2, "never"],
```

Comment:
ok ok, I'll add them back :) I can tell this is important to you, and I don't mind semicolons. However, I will go on record as saying that I think we're clinging to a Crockford rule that isn't really buying us anything in this day and age.